README.md

```diff
@@ -94,7 +95,7 @@ Spec: Free-tier Oracle VM - Amsterdam - 2.4Ghz quad-core ARM64 CPU, 24GB RAM
 | OAuth support                                | Yes    | Yes |
 | API Keys                                     | N/A    | Yes |
 | LivePhoto/MotionPhoto backup and playback    | Yes    | Yes |
-| Support 360 degree image display             | No     | Yes |
+| Support 360 degree image display             | Yes    | Yes |
```

Comment:
This is marking mobile as supporting panoramas, but the PR you linked just affected support for 360 videos on the web.

Reply:
my bad